src/states/Lobby.gd

```diff
@@ -0,0 +1,170 @@
+class_name Lobby
+extends Control
+
+
+class Sorter:
+	static func sort_by_name(a, b):
+		return a.name.to_lower() < b.name.to_lower()
+
+
+# The radius of the circle of buttons.
+const BUTTON_CIRCLE_RADIUS := 120
+# The size of each button.
+const BUTTON_SIZE := 40
+# The available colors.
+const COLORS := [
+	Color(1.0, 0.0, 0.0),
+	Color(1.0, 0.5, 0.0),
+	Color(1.0, 1.0, 0.0),
+	Color(0.0, 1.0, 0.0),
+	Color(0.0, 1.0, 1.0),
+	Color(0.0, 0.5, 1.0),
+	Color(0.5, 0.0, 1.0),
+	Color(1.0, 0.0, 1.0)
+]
+# The number of rows in the table
+const NUM_ROWS = 8
```

Comment:
Pretty sure you can do `const NUM_ROWS = len(COLORS)`, also should probably be named `NUM_COLORS`

Reply:
wait nevermind, you mean how many rows in the lobby table thingy lol